ExampleCode/C++/API/API.cpp

```diff
@@ -80,22 +81,38 @@ void iSENSE::set_contributor_key(string proj_key)
 }
 
 
-// Sets the email address to be used to upload data
-void iSENSE::set_email(string proj_email)
+// Users should never have to call this method, as it is possible to
+// pull datasets and compare dataset names to get the dataset_ID
+// Users should instead use the append byName method.
+void iSENSE::set_dataset_ID(string proj_dataset_ID)
 {
-  email = proj_email;
+  dataset_ID = proj_dataset_ID;
 }
 
 
-// Sets the password to be used
-void iSENSE::set_password(string proj_password)
+// Sets both email & password at once. Checks for valid email / password.
+bool iSENSE::set_email_password(string proj_email, string proj_password)
 {
+  email = proj_email;
   password = proj_password;
+
+  if(get_check_user())

```

Comment:
This method name doesn't make too much sense to me: what does `get_check_user()` do?